delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -769,12 +857,25 @@ object Files {
       case s: Current                   => IOUtils.instant.map(FileDeprecated(c.id, c.project, s.rev + 1L, _, c.subject))
     }
 
+    def migrate(c: MigrateFile) = state match {
```

Comment:
Same here, the migration code is being entangled with the production regular code